src/cow-react/api/gnosisSafe/hooks/useSafeApiClient.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react'
+import { useWalletInfo } from '@cow/modules/wallet'
+import { useWeb3React } from '@web3-react/core'
+import { createSafeServiceClient } from '@cow/api/gnosisSafe'
+import SafeApiKit from '@safe-global/api-kit'
+
+export function useSafeApiClient(): SafeApiKit | null {
+  const [safeApiClient, setSafeApiClient] = useState<SafeApiKit | null>(null)
+  const { chainId } = useWalletInfo()
+  const { provider } = useWeb3React()
+
+  useEffect(() => {
```

Comment:
why for the `safeSdk` hook you check for `isSafeWallet` and here you don't?

Reply:
Because you don't need to be inside a Safe to call the Safe api, that's all.
But I guess it makes sense, will update